libqtile/window.py

```diff
@@ -1316,3 +1316,14 @@ def cmd_set_position(self, dx, dy, curx, cury):
                 self.group.layout.focused = index2
                 self.group.layoutAll()
                 break
+
+    def __getstate__(self):
+        d = {}
+        d['group'] = self.group.name
+        d['window'] = self.window.wid
+        return d
+
+    def __setstate__(self, state):
+        self.Wid = state['window']
```

Comment:
In fact, since these are coming from self.group.name and going to self.gid, I don't understand how/why this works.